src/workerd/server/workerd-api.c++

```diff
@@ -312,6 +315,10 @@ kj::Own<jsg::ModuleRegistry> WorkerdApiIsolate::compileModules(
     }
   }
 
+  if (featureFlags.getNodeJsCompat()) {
+    modules->addBuiltinBundle(NODE_BUNDLE);
+  }
```

Comment:
See https://github.com/cloudflare/workerd/pull/208 ... there I add a `workerd/api/node/node.h` that will contain the method for registering all of the node.js compat stuff. Whichever of these PRs lands first, the other should be updated so that this bundle registration happens in there.

Reply:
Ack. I'll either rebase mine or will fix it once your change is in. I'd rather not create a node.h in this PR too to prevent merge conflicts.